libs/printing/src/render.tsx

```diff
@@ -58,27 +58,39 @@ function getContentHeight(page: Page): Promise<number> {
   });
 }
 
+export type DocumentRenderer = () => JSX.Element | JSX.Element[];
+
 export interface RenderSpec {
-  document: JSX.Element | JSX.Element[];
+  document: JSX.Element | JSX.Element[] | DocumentRenderer;
```

Comment:
What is the rationale for adding deferred rendering rather than just passing an element from `design/backend`? I think it's fine but don't see why it's necessary.

Reply:
Ah, leftover from a previous approach that I don't need anymore -- I tried removing the `<ThemeProvider>` wrapper from one of the existing reports earlier and tests were failing due to missing theme context and I mistakenly thought the components were getting mounted somewhere else before getting used here, but the actual fix was just to move [this line](https://github.com/votingworks/vxsuite/pull/4917/files#diff-14c71a93ae4f41c491a64dd5750495b30d6b47331a893fd4c2a2734c5588790cL117) up into the theme wrapper. Will revert this